flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkDistributionMode.java

```diff
@@ -81,13 +81,20 @@ public class TestFlinkIcebergSinkDistributionMode extends TestFlinkIcebergSinkBa
   @Parameter(index = 1)
   private boolean partitioned;
 
-  @Parameters(name = "parallelism = {0}, partitioned = {1}")
+  @Parameter(index = 2)
+  private boolean useV2Sink;
+
+  @Parameters(name = "parallelism = {0}, partitioned = {1}, useV2Sink = {3}")
```

Comment:
Cut the number of tests, like above

Reply:
I eventually hardcoded it to `DO_NOT_USE_V2_SINK` because in some tests there were used `rangeDistributionStatisticsType` which is not available in `IcebergSink`.